cypress/integration/projectSettings/project_settings.ts

```diff
@@ -457,45 +461,57 @@ describe("Project Settings when defaulting to repo", () => {
     cy.preserveCookies();
   });
 
-  it("Should not have the save button enabled on load", () => {
-    cy.dataCy("save-settings-button").should("be.disabled");
-  });
+  describe("General Settings page", () => {
+    it("Should not have the save button enabled on load", () => {
+      cy.dataCy("save-settings-button").should("be.disabled");
+    });
 
-  it("Preserves edits to the form when navigating between settings tabs and does not show a warning modal", () => {
-    cy.dataCy("spawn-host-input").should("have.value", "/path");
-    cy.dataCy("spawn-host-input").type("/test");
-    cy.dataCy("save-settings-button").should("not.be.disabled");
-    cy.dataCy("navitem-access").click();
-    cy.dataCy("navigation-warning-modal").should("not.be.visible");
-    cy.dataCy("navitem-general").click();
-    cy.dataCy("spawn-host-input").should("have.value", "/path/test");
-  });
+    it("Preserves edits to the form when navigating between settings tabs and does not show a warning modal", () => {
+      cy.dataCy("spawn-host-input").should("have.value", "/path");
+      cy.dataCy("spawn-host-input").type("/test");
+      cy.dataCy("save-settings-button").should("not.be.disabled");
+      cy.dataCy("navitem-access").click();
+      cy.dataCy("navigation-warning-modal").should("not.be.visible");
+      cy.dataCy("navitem-general").click();
+      cy.dataCy("spawn-host-input").should("have.value", "/path/test");
+    });
 
-  it("Enables the save button", () => {
-    cy.dataCy("save-settings-button").should("not.be.disabled");
-  });
+    it("Enables the save button", () => {
+      cy.dataCy("save-settings-button").should("not.be.disabled");
+    });
 
-  it("Shows a 'Default to Repo' button on page", () => {
-    cy.dataCy("default-to-repo-button").should("exist");
-  });
+    it("Shows a 'Default to Repo' button on page", () => {
+      cy.dataCy("default-to-repo-button").should("exist");
+    });
 
-  it("Shows a third radio box when rendering a project that inherits from repo", () => {
-    cy.dataCy("enabled-radio-box").children().should("have.length", 3);
-  });
+    it("Shows a third radio box when rendering a project that inherits from repo", () => {
+      cy.dataCy("enabled-radio-box").children().should("have.length", 3);
+    });
 
-  it("Does not default to repo value for display name", () => {
-    cy.dataCy("display-name-input").should("not.have.attr", "placeholder");
-  });
+    it("Does not default to repo value for display name", () => {
+      cy.dataCy("display-name-input").should("not.have.attr", "placeholder");
+    });
 
-  it("Shows a navigation warning modal when clicking on a header link", () => {
-    cy.get("a[href='/user/patches']").click();
-    cy.dataCy("navigation-warning-modal").should("be.visible");
-    cy.get("body").type("{esc}");
-  });
+    it("Shows a navigation warning modal when clicking on a header link", () => {
```

Comment:
nit is this behavior exclusive to clicking a header link or navigating away in general?
if the latter maybe just giving it a more generic test name.